Billing.API/Services/Invoice/InvoiceProviderOptions.cs

```diff
@@ -3,10 +3,9 @@ namespace Billing.API.Services.Invoice
     public class InvoiceProviderOptions
     {
         public bool UseDummyData { get; set; }
-        public string Host { get; set; } = string.Empty;
-        public string Username { get; set; } = string.Empty;
-        public string Password { get; set; } = string.Empty;
         public string Schema { get; set; } = string.Empty;
         public string DbConnectionString { get; set; } = string.Empty;
+        public string HashKey { get; set; } = string.Empty;
```

Comment:
What do you think about something more descriptive? I am not sure, maybe:

```suggestion
        public string SignatureHashKey { get; set; } = string.Empty;
``` 


```suggestion
        public string PublicSignatureHashKey { get; set; } = string.Empty;
```